apps/meteor/client/views/navigation/sidepanel/SidePanelOmnichannelBadges/SidePanelOmnichannelBadges.tsx

```diff
@@ -0,0 +1,23 @@
+import type { IRoom, ISubscription } from '@rocket.chat/core-typings';
+import { isOmnichannelRoom } from '@rocket.chat/core-typings';
+
+import SidePanelPriorityTag from './SidePanelPriorityTag';
+import { RoomActivityIcon } from '../../../../omnichannel/components/RoomActivityIcon';
+import { useOmnichannelPriorities } from '../../../../omnichannel/hooks/useOmnichannelPriorities';
+
+const SidePanelOmnichannelBadges = ({ room }: { room: ISubscription & IRoom }) => {
+	const { enabled: isPriorityEnabled } = useOmnichannelPriorities();
+
+	if (!isOmnichannelRoom(room)) {
```

Comment:
👀

Reply:
removing because it's being checked in the parent component